roles/ncn-common-install/defaults/main.yml

```diff
@@ -16,7 +22,7 @@ services_common:
     enabled: yes
     state: stopped
   - name: kdump-early.service
-    enabled: yes
+    enabled: no
```

Comment:
Change back to yes and change the service back to started for kdump and kdump early.

Reply:
don't forget the tests